### ISSUE TITLE
[E2E] Epinio install on RD

### ISSUE THREAD

**Issue by prabalsharma** [rancher-sandbox/rancher-desktop#885]
Expand E2E test to validate epinio installation on RD

Test case:
Heads up: Epinio has a new install method - helm chart
https://docs.epinio.io/installation/install_epinio_auto.html?highlight=helm#deploy-epinio
1. Deploy RD on all supported platforms (E2E CI is already doing this)
2. Deploy epinio; latest build (an option to specify epinio build will be a nice to have and can be added on later when required)
3. Push a sample app on epinio

As we know that app push on epinio is not working on RD-on-macos, we can let E2E CI ignore this failure and continue with rest of the tests.

**Comment by evertonlperes** [comment]
Filed an issue to investigate Epinio Install on RD (macOS): https://github.com/epinio/epinio/issues/946#issuecomment-954038430

**Comment by evertonlperes** [comment]
Depends on: #936

**Comment by evertonlperes** [comment]
Tagging: https://github.com/epinio/epinio/issues/1116

**Comment by ericpromislow** [comment]
Replaced by #1774 (because Everton left the team and company).

**Comment by ericpromislow** [comment]
Accidentally closed this when I closed the old PR.

**Comment by jandubois** [comment]
I think this was supposed to be closed and not re-opened.
